controllers/auction.js

```diff
@@ -49,6 +50,12 @@ const createNewAuction = async (req, res) => {
   try {
     const { id: seller } = req.userData
     const { initial_price: initialPrice, item_type: itemType, end_time: endTime, quantity } = req.body
+
+    const { currencies } = await wallet.fetchWallet(seller)
+    const itemQuantity = parseInt(currencies[`${itemType}`])
+    if (!itemQuantity) return res.boom.forbidden(`You do not have any ${itemType}!`)
```

Comment:
Rather than explicitly showing the user does not have any coins, we can simply say that `You do not have enough of ${itemType}s!` to avoid showing explicit details.

We can return that message for `if(!itemQuantity || itemQuantity < quantity)`

Reply:
Done.